crex/models/scheduled_course_section.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+from typing import Tuple
+from crex.models import CourseSection
+from dataclasses import dataclass
+from dataclasses_json import dataclass_json
+from rdflib import URIRef
+
+
+@dataclass_json
+@dataclass(frozen=True)
+class ScheduledCourseSection(CourseSection):
+    semester: str
+    schedule: str  # TODO this maybe should be like, a date time? definitely not a string.
```

Comment:
maybe it'll end up being best to have schedule stuff handled by other apps e.g. yacs